crates/storage/nippy-jar/src/lib.rs

```diff
@@ -88,18 +82,22 @@ pub struct NippyJar<H = ()> {
     user_header: H,
     /// Number of data columns in the jar.
     columns: usize,
+    /// Number of data rows in the jar.
+    rows: usize,
     /// Optional compression algorithm applied to the data.
     compressor: Option<Compressors>,
+    #[serde(skip)]
     /// Optional filter function for data membership checks.
     filter: Option<InclusionFilters>,
+    #[serde(skip)]
     /// Optional Perfect Hashing Function (PHF) for unique offset mapping.
     phf: Option<Functions>,
     /// Index mapping PHF output to value offsets in `offsets`.
     #[serde(skip)]
     offsets_index: PrefixSummedEliasFano,
     /// Offsets within the file for each column value, arranged by row and column.
     #[serde(skip)]
-    offsets: EliasFano,
+    offsets: Vec<u64>,
```

Comment:
next PR actually moves this field entirely from here (https://github.com/paradigmxyz/reth/pull/5669) ,

however, the goal here (and the PR), is to only use this field when building the snapshot, since loading it is too expensive since we'd need to load the whole EliasFano struct (eg. RPC call)

Reply:
and do the reading of rows/columns through`DataReader` which reads singular offsets directly from disk, without loading the whole thing (as EF)